src/builder/paccache.rs

```diff
@@ -0,0 +1,46 @@
+use crate::internal::{commands::ShellCommand, error::AppResult};
+
+#[derive(Debug, Default)]
+pub struct PaccacheBuilder {
+    keep: i32,
+    keep_ins: bool,
+    quiet: bool,
+}
+
+impl PaccacheBuilder {
+    pub fn keep(mut self, keep: i32) -> Self {
```

Comment:
i would change this function name to something like `set_keep` to make it more obvious what it does, same thing for keep_ins and quiet

Reply:
do you think something like `keep_ins_pkgs` is good? because that is setting an option to keep installed packages in the pacman cache.

Reply:
yes that works too